pkg/cmd/pr/list/list.go

```diff
@@ -163,6 +168,14 @@ func listRun(opts *ListOptions) error {
 		return opts.Browser.Browse(openURL)
 	}
 
+	if opts.App != "" {
+		if filters.Author != "" {
+			return fmt.Errorf("--app can't be specified when --author is specified")
```

Comment:
This check should be performed in RunE block where we typically validate flags, could use the `cmdutil.MutuallyExclusive()` helper, or should alternatively return an error via `cmdutil.FlagErrorf()`

Reply:
https://github.com/cli/cli/pull/5180/commits/2809a2e16cd9bbff0d9746c4355fbfca261a400b